ArduPlane/ArduPlane.cpp

```diff
@@ -968,6 +969,81 @@ void Plane::update_flight_stage(void)
     airspeed.set_EAS2TAS(barometer.get_EAS2TAS());
 }
 
+/*
+*  ArduSoar support function
+*
+*  Peter Braswell and Samuel Tabor
+*/
+void Plane::update_soaring() {
+    
+    if (!soaring_controller.is_active()) {
+        return;
```

Comment:
Added the initialization of private member variables to SoaringController's constructor.